embulk-input-s3/src/main/java/org/embulk/input/s3/HttpProxy.java

```diff
@@ -14,15 +14,15 @@
 {
     private final String host;
     private final Optional<Integer> port;
-    private final boolean https;
+    private final Optional<Boolean> https;
```

Comment:
Do we need this to be `Optional`? Don't we have a kind of `Default` options there?